### PR TITLE
Enable preferreduilang on Desktop compiler.  And add it to buildtask

### PR DESCRIPTION
**Pull request by KevinRansom**
The desktop compiler didn't have this option.
The buildtask didn't allow it to be set easily.

This PR addresses that.

Needed for localization of dotnet cli.

+minor tweak to .gitignore

Fixes: https://github.com/Microsoft/visualfsharp/issues/3272

### CODE REVIEW

**Comment by KevinRansom** [comment]
@Pilchie @cartermp @brettfo We will need to cherry pick this.

File: tests/fsharpqa/Source/CompilerOptions/fsc/help/help40.437.1033.bsl [hunk 1]
```diff
@@ -94,6 +94,7 @@ Copyright (c) Microsoft Corporation. All Rights Reserved.
 		- ADVANCED -
 --codepage:<n>                           Specify the codepage used to read source files
 --utf8output                             Output messages in UTF-8 encoding
+--preferreduilang:<string>               Specify a strong name key container
```
Comment (brjackson) [inline tests/fsharpqa/Source/CompilerOptions/fsc/help/help40.437.1033.bsl]: Copy paste error?

File: tests/fsharpqa/Source/CompilerOptions/fsi/exename/help40.437.1033.bsl [hunk 2]
```diff
@@ -51,6 +51,7 @@ Usage: fsharpi <options> [script.fsx [<arguments>]]
 		- ADVANCED -
 --codepage:<n>                           Specify the codepage used to read source files
 --utf8output                             Output messages in UTF-8 encoding
+--preferreduilang:<string>               Specify a strong name key container
```
Comment (brjackson) [inline tests/fsharpqa/Source/CompilerOptions/fsi/exename/help40.437.1033.bsl]: Copy paste error?

File: tests/fsharpqa/Source/CompilerOptions/fsi/help/help40-nologo.437.1033.bsl [hunk 3]
```diff
@@ -51,6 +51,7 @@ Usage: fsi.exe <options> [script.fsx [<arguments>]]
 		- ADVANCED -
 --codepage:<n>                           Specify the codepage used to read source files
 --utf8output                             Output messages in UTF-8 encoding
+--preferreduilang:<string>               Specify a strong name key container
```
Comment (brjackson) [inline tests/fsharpqa/Source/CompilerOptions/fsi/help/help40-nologo.437.1033.bsl]: Copy paste error?

File: tests/fsharpqa/Source/CompilerOptions/fsi/help/help40.437.1033.bsl [hunk 4]
```diff
@@ -53,6 +53,7 @@ Usage: fsi.exe <options> [script.fsx [<arguments>]]
 		- ADVANCED -
 --codepage:<n>                           Specify the codepage used to read source files
 --utf8output                             Output messages in UTF-8 encoding
+--preferreduilang:<string>               Specify a strong name key container
```
Comment (brjackson) [inline tests/fsharpqa/Source/CompilerOptions/fsi/help/help40.437.1033.bsl]: Copy paste error?

File: tests/fsharpqa/Source/CompilerOptions/fsc/help/help40.437.1033.bsl (hunk 1 above)
Comment (KevinRansom) [inline tests/fsharpqa/Source/CompilerOptions/fsc/help/help40.437.1033.bsl]: @brjackson ... thanks.

**Comment by nguerrera** [comment]
Thanks for the quick turnaround on this!

**Pull request merged**